lib/smart_proxy_remote_execution_ssh/runners/script_runner.rb

```diff
@@ -160,6 +164,16 @@ def trigger(*args)
       run_async(*args)
     end
 
+    def detect_capabilities
```

Comment:
It implies multiple different capabilities, but this method only checks for `unshare` support :/

Reply:
Eventually there might be more, but I can rename it if you feel strongly about it